image/cli-base/install/install-ibmcloud.sh

```diff
@@ -25,12 +25,17 @@ fi
 
 CLI_VERSION=2.30.0
 wget -q https://download.clis.cloud.ibm.com/ibm-cloud-cli/${CLI_VERSION}/IBM_Cloud_CLI_${CLI_VERSION}_${TARGET_PLATFORM}.tar.gz
+
 tar -xzf IBM_Cloud_CLI_${CLI_VERSION}_${TARGET_PLATFORM}.tar.gz
 mv Bluemix_CLI/bin/ibmcloud /usr/local/bin/
 rm -rf Bluemix_CLI IBM_Cloud_CLI_${CLI_VERSION}_${TARGET_PLATFORM}.tar.gz
 ibmcloud plugin repo-plugins -r 'IBM Cloud'
-ibmcloud plugin install container-service
 ibmcloud plugin install container-registry
+#Plugin not supported in ppc64le (Could not find compatible binary to install for plug-in container-service[kubernetes-service/ks]
```

Comment:
we should update the readme so it's clear that the container service plugin is not available on power

Reply:
updated in readme